src/synchronizeContacts.js

```diff
@@ -1,8 +1,15 @@
 const get = require('lodash/get')
-const merge = require('lodash/merge')
 const pLimit = require('p-limit')
+const mergeWith = require('lodash/mergeWith')
+const isArray = require('lodash/isArray')
 const transpileContactToCozy = require('./helpers/transpileContactToCozy')
 
+const customizer = (objValue, srcValue) => {
```

Comment:
do we need objValue ? I don't understand what this function should do

Reply:
Can we add jsdocs ?

Reply:
Yep, thanks for the feedback, this portion was not great. I rewrote it to be more specific and explicit, let me know what you think.